docs/arch_features.md

```diff
@@ -43,7 +43,6 @@ For EC2 specifically, there are a few additional steps:
 
 1. If connecting to EC2 instance via remote window, the toolkit generates temporary SSH keys (30 second lifetime), with the public key sent to the remote instance.
```

Comment:
Should we also mention here the expected remote user name/home dir? That is a detail that may be useful for troubleshooting.

Reply:
good idea, I also added a note about which OS are supported atm.